### DISCUSSION TITLE
Separte comments from name (article/service) column in PDF invoice and e.g. sales orders

### DISCUSSION THREAD

**Discussion by blu-IT** [YetiForceCompany/YetiForceCRM#17337] [How to]
Hi,

I am working ob PDF templates and noticed, that a comment (red boxes) is printed inside the name column, when generating the PDF:

![grafik](https://github.com/YetiForceCompany/YetiForceCRM/assets/13665806/d65e7a63-d764-4de8-8b7e-e221501e1e26)

What i need is, that the comment is only displayed in the comments column (Beschreibung) and **not in the name column**.

The rows are generated with this piece of code in my sales orders:

`$(inventory : type=table columns=seq,name,comment1,value10 , href=no , labels=SNo,Artikel,Beschreibung , width=5%,20%,40%)$`

Any idea?

Thanks!

**Comment by rskrzypczak** [comment]
Two solutions come to mind:
1. Add a new text field (Text) to the advanced block and do not use the wysiwyg field.
2. Write your own parsing function.
